contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,133 @@
+
+## Optimization cheat sheet
+If you're looking to improve any of the above metrics, here is a handy cheat sheet for where to look for opportunities. Each value in the table indicates how likely the metrics are to be affected.
+
+| What?   | First Contentful Paint  | DOM Interactive | Page Loaded |
+|--------|--------|--------|--------|
+| Page Size   | 🟢 High  | 🟢 High   | 🟢 High |
+| Server Response Time   | 🟢 High  | 🟢 High | 🟢 High |
+| Network Latency   | 🟢 High  | 🟢 High | 🟢 High |
+| Downloading JavaScript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+| Executing JavaScript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+|API Requests |	🟡 Medium (only if they block rendering or are critical for initial content) |	🟡 Medium (only if they are for rendering critical content) |	🟢 High |
+| CSS Files: Downloading and Parsing  | 🟢 High | 🟡 Medium | 🟡 Medium |
+| Images: Downloading and Rendering| 🟡 Medium (only if they are part of the initial content) | 🔴 Low | 🟢 High |
+| Fonts: Downloading and Rendering  | 🟡 Medium (only if they are part of the initial content)  | 🔴 Low |  🔴 Low  |
+
+## Identifying opportunities on your own site
+PostHog's performance monitoring tool shows you the size and speed of different requests made during page load. You can also filter these by request type, such as API, JS, image, and CSS requests.
+
+Combine this with the cheat sheet above and you'll be able to pinpoint what are the contributing factors to your metric times.
+
+### **Examples**
+**1. API requests**
+![A screenshot of the network tab filtered to show API requests only. One API request is identified as taking too long.](https://i.imgur.com/JPeZeTe.png)
+
+Here we can see an API request that is happening during page load (indicated by the `LOAD` text next to it). Since it is blocking the initial rendering, it is contributing to the high FCP time as well as the Page Loaded time.
+
+**2. JS requests**
+
+![A screenshot of the network tab filtered to show JavaScript requests only](https://i.imgur.com/klPHLwK.png)
+In this example we see fast times for downloading JavaScript. Expanding into one of the requests, we can also see that the request is not blocking the render of the screen. Thus we can conclude that download JS is not contributing significantly to any of our metrics.
+
+**3. Image requests**
+![A screenshot of the network tab filtered to show image requests only. The list shows small images that are taking more than one second each to download.](https://i.imgur.com/4b45d2k.png)
+
+This example shows tiny images that are taking in excess of one second to download. This may be caused by server or network latency.
+
+## Turning opportunities into results
+
+Once you have identified which factors are affected your page load times, here are some actionable next steps to improve them:
+
+### JavaScript
```

Comment:
Remove unused libraries, packages, dependencies, or change to less intense ones.